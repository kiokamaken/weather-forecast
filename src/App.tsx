import React from 'react';
import Router from 'router';
import { Provider } from 'react-redux';
import { ThemeProvider } from 'styled-components';
import { light } from 'components/Layout/Theme';
import GlobalStyle from 'globalStyles';
import store from './store';

function App() {
  return (
    <Provider store={store}>
      <ThemeProvider theme={light}>
        <GlobalStyle />
        <Router />
      </ThemeProvider>
    </Provider>
  );
}

export default App;
