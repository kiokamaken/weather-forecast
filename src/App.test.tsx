import React from 'react';
import { shallow } from 'enzyme';
import App from './App';

describe('App Test Suite', () => {
  it('Should render without crashing', () => {
    const app = shallow(<App />);
    expect(app).toMatchSnapshot();
  });
});
