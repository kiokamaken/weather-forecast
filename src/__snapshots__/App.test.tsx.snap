// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App Test Suite Should render without crashing 1`] = `ShallowWrapper {}`;
